spec/amadeus/namespaces.test.js

```diff
@@ -146,16 +146,23 @@ describe('Namespaces', () => {
         .toHaveBeenCalledWith('/v1/reference-data/locations/airports', {});
     });
 
-    it('.amadeus.referenceData.locations.pointsOfInterest.get', () => {
+    it('.amadeus.referenceData.locations.pointsOfInterest().get', () => {
       amadeus.client.get = jest.fn();
-      amadeus.referenceData.locations.pointsOfInterest.get();
+      amadeus.referenceData.locations.pointsOfInterest().get();
       expect(amadeus.client.get)
         .toHaveBeenCalledWith('/v1/reference-data/locations/pois', {});
     });
 
-    it('.amadeus.referenceData.locations.pointsOfInterest.bySquare.get', () => {
+    it('.amadeus.referenceData.locations.pointsOfInterest(\'XXX\').get', () => {
```

Comment:
Can you replace it by a real example?

Why do we need a \ here?

Reply:
I could replace \' with ".